### ISSUE TITLE
Error on upgrading

### ISSUE THREAD

**Issue by CaptainQuirk** [vim-vdebug/vdebug#474]
Upon upgrading vdebug today, I suddenly get this error : 

```
ligne  114 :
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/lmessier/.vim/bundle/vdebug/python3/vdebug/debugger_interface.py", line 4, in <module>
    from . import event
  File "/home/lmessier/.vim/bundle/vdebug/python3/vdebug/event.py", line 12, in <module>
    from .ui import vimui
  File "/home/lmessier/.vim/bundle/vdebug/python3/vdebug/ui/vimui.py", line 349, in <module>
    class SourceWindow(interface.Window):
  File "/home/lmessier/.vim/bundle/vdebug/python3/vdebug/ui/vimui.py", line 355, in SourceWindow
    or vim.funcs.has('nvim-0-4-0')
AttributeError: module 'vim' has no attribute 'funcs'
ligne  115 :
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'vdebug' is not defined
Erreur détectée en traitant function Vdebug_load_options :
ligne    9 :
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'debugger' is not defined
Erreur détectée en traitant function Vdebug_load_keymaps :
ligne   27 :
Traceback (most recent call last):
  File "<string>", line 1, in <module>
NameError: name 'debugger' is not defined
```

I use vim **8.1** on **Linux Mint 20** an I have **python 3.8**

The problem goes away if I do a hard reset on `d914bcb`

**Comment by BlackIkeEagle** [comment]
Sorry, it seems this is only available in neovim, I'll fix it

**Comment by BlackIkeEagle** [comment]
@CaptainQuirk could you verify the proposed fix works #475 ?

**Comment by BlackIkeEagle** [comment]
@CaptainQuirk I assumed the thumbsup is that it works :)
